src/libponyc/options/options.h

```diff
@@ -0,0 +1,19 @@
+#ifndef PONYC_OPTIONS_H
+#define PONYC_OPTIONS_H
+
+#include "../libponyc/pass/pass.h"
+#include "../libponyrt/options/options.h"
+
+#include <stdint.h>
+#include <stdbool.h>
+
+opt_arg_t* ponyc_opt_std_args(void);
+
+void ponyc_print_std_usage();
+
+bool ponyc_opt_process(opt_state_t* s, pass_opt_t* opt,
```

Comment:
By convention, I'd prefer to see the function names in this file all begin with a common prefix that matches the filename.  For example, this function would be called `options_process`.

Following this convention makes it really easy to find (and even guess) where a particular function is declared and implemented.

Reply:
Whoops, I forgot to delete it, its gone now.